apps/web/modules/d/[link]/d-type-view.tsx

```diff
@@ -35,6 +36,7 @@ export default function Type({
         entity={entity}
         duration={duration}
         hashedLink={hashedLink}
+        durationConfig={durationConfig}
```

Comment:
I don't see any use case for it

Reply:
@anikdhabal Replied above. We are passing the duration config to the `Booker` component also, so that the duration param passed to the private links are also respected.